packages/react/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -53,12 +53,15 @@ const chatMessageStyles: ComponentSlotStylesInput<
 
     ...getBorderFocusStyles({ siteVariables, isFromKeyboard: p.isFromKeyboard }),
 
-    ':hover': {
-      [`& .${ChatMessage.slotClassNames.actionMenu}`]: {
-        opacity: 1,
-        width: 'auto',
+    ...(v.showActionMenu === undefined && {
```

Comment:
to make logic more robust, would suggest to capture additional cases where `v.showActionMenu` may be nullish, by either:
```js
v.showActionMenu == null
```
or rely on lodash:
```js
_.isNil(v.showActionMenu)
```

Reply:
Added